src/index/api.rs

```diff
@@ -70,11 +68,8 @@ fn get_release_time_yanked_downloads(pkg: &MetadataPackage) -> Result<(time::Tim
                 .get("created_at")
                 .and_then(|c| c.as_str())
                 .ok_or_else(|| err_msg("Not a JSON object"))?;
-            release_time = Some(
-                time::strptime(release_time_raw, "%Y-%m-%dT%H:%M:%S")
-                    .unwrap()
-                    .to_timespec(),
-            );
+            release_time =
+                Some(NaiveDateTime::parse_from_str(release_time_raw, "%Y-%m-%dT%H:%M:%S").unwrap());
```

Comment:
In fact, this should be something like

```suggestion
                Some(NaiveDateTime::parse_from_str(release_time_raw, "%Y-%m-%dT%H:%M:%S%.f%:z")?);
```

in case the crates.io API returns an invalid value